BIT285_SpicyShrimps/BIT285_SpicyShrimps/Controllers/LoginController.cs

```diff
@@ -27,19 +27,54 @@ public ActionResult Index()
             return View("Index");
         }
 
-        //teacher login Get method
+       
+
+        //student login Get methods
+        [HttpGet]
+        public ActionResult studentLogin()
+        {
+            return View("StudentLogin");
+        }
+
+        [HttpGet]
+        public ActionResult VerifyTeacher()
+        {
+            return View("VerifyTeacher");
+        }
+
+        [HttpGet]
+        public ActionResult ChooseName()
+        {
+            return View("ChooseName");
+        }
+
+        [HttpGet]
+        public ActionResult SecretWord()
+        {
+            return View("SecretWord");
+        }
+
+        //teacher login Get methods
         [HttpGet]
         public ActionResult teacherLogin()
         {
             return View("TeacherLogin");
         }
 
-        //student login Get method
         [HttpGet]
-        public ActionResult studentLogin()
```

Comment:
naming convention of ActionResult methods are inconsistent, some start with lower case ie studentLogin